Makefile

```diff
@@ -16,3 +16,7 @@ $(BUILDDIR):
 
 clean:
 	rm -r $(BUILDDIR)/
+
+.PHONY: check
+check: all
+	cd $(BUILDDIR); tox -c ..  -- -v --maxfail=1
```

Comment:
Can you add `-m 'not exclude_from_coverage'` to the pytest arguments? (i.e. after the `--`). The `exclude_from_coverage` take extra long. Before running tox, a couple of other commands are:
```
flake8
ci/check-using-std.sh
```